Minecraft Neoforged template (sucessor to Forge)

spusuf opened the pull request:
Have stress tested with 8 friends over 2 months. No issues.

spusuf committed:
Create minecraft-neoforged.json

spusuf committed:
Create README.md

elBelgg commented:
Tested it, working good.

spusuf commented:
> Tested it, working good.

I use this for spinning up multiple servers, so I know it works on my machines. But if there are any changes needed I'm happy to write them in.

kris545545 commented:
1.20 Neoforge builds are downloaded from `https://maven.neoforged.net/releases/net/neoforged/forge/` not `https://maven.neoforged.net/releases/net/neoforged/neoforge/`

It's the same naming scheme with the libraries

spusuf committed:
Merge branch 'pufferpanel:v2.6' into v2.6

spusuf committed:
Add depotdownloader workaround

Adds depotdownloader workaround to fix server unable to start, and added config currency date

spusuf committed:
Updated config file + follows default order

spusuf committed:
Fix brackets

spusuf committed:
Update README.md
